src/Test/winswTests/winswTests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net40;net461;netcoreapp3.1</TargetFrameworks>
```

Comment:
I am still waiting to see a prototype of how it resolves the .NET 4.0 support requirements in the binaries

Reply:
Update in https://github.com/winsw/winsw/pull/547/commits/cfacc38720922d1068f4334cba1c81936c069d62. Review will help to proceed!

Reply:
@buddhikac96 We still support net40 in v2.